src/Map/WeatherTileResourceProvider_P.h

```diff
@@ -111,6 +111,8 @@ namespace OsmAnd
         mutable QReadWriteLock _lock;
         int _priority;
         int _obtainValuePriority;
+        int _currentDownloadingsCountToObtainValue;
```

Comment:
It's better to have list of downloading and evaluating tiles like `QList<TileId>`